### PR TITLE
Add new rosa jobs and add contacts for rosacli e2e test

### PR DESCRIPTION
**Pull request by yasun1**
[OCPQE-17919](https://issues.redhat.com//browse/OCPQE-17919)

### CODE REVIEW

**Comment by yasun1** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-master-rosacli-aws-rosa-hcp-e2e-f3 periodic-ci-openshift-openshift-tests-private-release-4.15-amd64-nightly-aws-rosa-non-sts-f7

**Comment by yasun1** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-master-rosacli-aws-rosa-hcp-e2e-f3 periodic-ci-openshift-openshift-tests-private-release-4.15-amd64-nightly-aws-rosa-non-sts-f7

**Comment by yasun1** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-master-rosacli-aws-rosa-hcp-e2e-f3 periodic-ci-openshift-openshift-tests-private-release-4.15-amd64-nightly-aws-rosa-non-sts-f7

**Comment by liangxia** [comment]
It's better to ask sub-team to review the changes first. 
To be honest, I do not have the knowledge to understand most of the changes in rosacli

File: ci-operator/jobs/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15-periodics.yaml [hunk 1]
```diff
@@ -17281,17 +17281,7 @@ periodics:
...
-    slack:
-      channel: '#ocm-qe-prow-ci-jobs'
-      job_states_to_report:
-      - failure
-      - error
-      report_template: '{{if eq .Status.State "success"}} :rainbow: Job *{{.Spec.Job}}*
-        ended with *{{.Status.State}}*. <{{.Status.URL}}|View logs> :rainbow: {{else}}
-        :volcano: Job *{{.Spec.Job}}* ended with *{{.Status.State}}*. <{{.Status.URL}}|View
-        logs> :volcano: {{end}}'
+  name: periodic-ci-openshift-openshift-tests-private-release-4.15-amd64-nightly-aws-rosa-non-sts-f7
```
Comment (liangxia) [inline ci-operator/jobs/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15-periodics.yaml:17284]: The `reporter_config` is removed. Is it expected ?
Also the issue happens in 4.14, too

Comment (yasun1) [reply ci-operator/jobs/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15-periodics.yaml:17284]: Update accordingly. Please review it again. Thanks.

**Pull request merged**